src/lib/isGitHub.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Small env check to determine if we're in a GitHub Actions environment
+ * @link https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+ */
+module.exports = function isGHA() {
+  return process.env.GITHUB_ACTIONS === 'true';
```

Comment:
We can replace this with the [ci-info](https://npm.im/ci-info) library I'm pulling in for RM-3621 to do `require('ci-info').GITHUB_ACTIONS`. That checks the `GITHUB_ACTIONS` env variable and also has awareness if it's part of a pull request.

https://github.com/watson/ci-info/blob/master/vendors.json#L93-L96